schemas/towns_fund.py

```diff
@@ -0,0 +1,278 @@
+        "columns": {
+            "Date": "datetime",
+            "Month": "int",
+            "Year": "str",
+            "FY": "str",
+            "FH": "int",
+            "FQ": "int",
+        },
+        "uniques": ["Date"],
+    },
+    "EndDate_Dim": {
+        "columns": {
+            "Date": "datetime",
+            "Month": "int",
+            "Year": "str",
+            "FY": "str",
+            "FH": "int",
+            "FQ": "int",
+        },
+        "uniques": ["Date"],
+    },
+    "Project_Dim": {
+        "columns": {
+            "Project_ID": "str",
+            "Project Name": "str",
+            "Package_ID": "str",
+            "Address/Postcode": "str",
+            "Secondary Organisation": "str",
+        },
+        "uniques": ["Project_ID"],
+        "foreign_keys": {
+            "Package_ID": {"parent_table": "Package_Dim", "parent_pk": "Package_ID"}
+        },
+    },
+    "Package_Dim": {
+        "columns": {
+            "Package_ID": "str",
+            "Package_Name": "str",
+            "FundType_ID": "str",
+            "Organisation": "str",
+            "Name Contact Email": "str",
+            "Project SRO Email": "str",
+            "CFO Email": "str",
+            "M&E Email": "str",
+        },
+        "uniques": ["Package_ID"],
+        "foreign_keys": {
+            "Organisation": {
+                "parent_table": "Organisation_Dim",
+                "parent_pk": "Organisation",
+            },
+            "Name Contact Email": {
+                "parent_table": "Contact_Dim",
+                "parent_pk": "Email Address",
+            },
+            "Project SRO Email": {
+                "parent_table": "Contact_Dim",
+                "parent_pk": "Email Address",
+            },
+            "CFO Email": {"parent_table": "Contact_Dim", "parent_pk": "Email Address"},
+            "M&E Email": {"parent_table": "Contact_Dim", "parent_pk": "Email Address"},
+        },
+    },
+    "Contact_Dim": {
+        "columns": {
+            "Email Address": "str",
+            "Contact Name": "str",
+            "Organisation": "str",
+            "Telephone": "str",
+        },
+        "uniques": ["Email Address"],
+        "foreign_keys": {
+            "Organisation": {
+                "parent_table": "Organisation_Dim",
+                "parent_pk": "Organisation",
+            }
+        },
+    },
+    "Organisation_Dim": {
+        "columns": {"Organisation": "str", "Geography": "str"},
+        "uniques": ["Organisation"],
+    },
+    "Project_Delivery_Plan": {
+        "columns": {
+            "Milestone": "str",
+            "Project_ID": "str",
+            "Start_Date": "datetime",
+            "Finish_Date": "datetime",
+            "Status": "str",
+            "Comments": "str",
+        },
+        "foreign_keys": {
+            "Project_ID": {"parent_table": "Project_Dim", "parent_pk": "Project_ID"}
+        },
+        "enums": {"Status": {str(e) for e in enums.StatusEnum}},
```

Comment:
should we move logic (ie the set generator) out of the schema and into schema.py? Since it's pretty much the same unpack operation repeated each time (and good practice to keep logic out of config-type files where possible)

Then these schema enum definitions will just list the enum object itself?